Documentation/coding-guidelines/coding-style.md

```diff
@@ -20,7 +20,7 @@ The general rule we follow is "use Visual Studio defaults".
    blank lines between members of a type.
 8. Avoid spurious free spaces.
    For example avoid `if (someVar == 0)...`, where the dots mark the spurious free spaces.
-   Consider enabling "View White Space (Ctrl+E, S)" if using Visual Studio to aid detection.
+   Consider enabling "View White Space (Ctrl+R, Ctrl+W)" if using Visual Studio to aid detection.
```

Comment:
Maybe add "in the default configuration".

If someone has different shortcuts configured, it won't match again :wink: